lib/internal/modules/helpers.js

```diff
@@ -238,7 +280,10 @@ function normalizeReferrerURL(referrer) {
   return new URL(referrer).href;
 }
 
-// For error messages only - used to check if ESM syntax is in use.
+/**
+ * For error messages only, check if ESM syntax is in use.
+ * @param {string} code
```

Comment:
returns boolean is missing

Reply:
Inferred.

<img width="505" alt="image" src="https://github.com/nodejs/node/assets/456802/b31d65a0-1fd7-483a-8e58-50aef37faca3">